prompts/prompts.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 const (
-	namePattern        = "^[a-zA-Z\\s,\\.'\\-pL]{1,64}$"
+	namePattern        = "^[a-zA-Z\\s,\\.'\\pL]{1,64}$"
```

Comment:
Actually, you know what... the `\-` was meaning to allow hyphenated surnames.

So we'll still want that.. it must have been affecting the unicode definition.